packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -46,53 +49,94 @@ export interface MedianRate {
   rate: BigNumber
 }
 
+export type CurrencyPairIdentifier = Branded<Address, 'PairIdentifier'>
+
+/**
+ * Used to construct the pair identifier from a pair label (e.g. CELO/BTC)
+ * The pair identifier needs to be a valid ethereum address, thus we
+ * truncate a keccak of the pair label.
+ * This function returns a branded type which can be fed into the wrapper.
+ * @param pair a string
+ */
+export const pairIdentifier = (pair: string): CurrencyPairIdentifier => {
+  return ensureLeading0x(
+    keccak256(pair)
+      .slice(0, 20)
+      .toString('hex')
+  ) as CurrencyPairIdentifier
+}
+
+/**
+ * This will act as an enum of common pairs.
+ * We can't use a straight enum because we want the value
+ * to be the branded type.
+ *
+ * E.g. usage: sortedOracles.getRates(CurrencyPair.CELOBTC)
+ */
+type defaultPairs = 'CELOBTC' | 'CELOUSD'
+export const CurrencyPair: Record<defaultPairs, ReportTarget> = {
+  CELOBTC: pairIdentifier('CELO/BTC'),
+  CELOUSD: CeloContract.StableToken,
```

Comment:
oh! Would this work? It could avoid the need for `toCurrencyPairIdentifier`. Though... maybe it's too unwieldy with `addressFor` being an async function.

```typescript
  CELOUSD: this.kit.registry.addressFor(CeloContract.StableToken)
  ```

Reply:
Yeah sadly you can't do that because it's in the global scope and there's no kit instance there and thus no registry to lookup that async value so yeah we're left with this solution where we're hiding that complexity under the covers.

Reply:
Ah, yeah, that makes sense.